Showcase-iOSTests/GeneratedShowcaseMocks.swift

```diff
@@ -1,9 +1,10 @@
-// MARK: - Mocks generated from file: Showcase-iOS/Firebase/Service/DataReferenceable.swift at 2018-04-25 08:55:34 +0000
+// MARK: - Mocks generated from file: Showcase-iOS/Firebase/Service/DataReferenceable.swift at 2018-04-25 10:43:15 +0000
 
 
 import Cuckoo
 @testable import Showcase_iOS
 
+import Firebase
```

Comment:
dont commit this file as it is causing merge conflict. every time the project builds , this file gets generated.